docs/man/zbm-kcl.8.rst

```diff
@@ -5,72 +5,117 @@ zbm-kcl
 
 OPTIONS
 =======
 
 **-a** *argument*
 
-  Append the value of *argument* to the kernel command line. The value of *argument* can be a simple variable name for Boolean arguments or may take the form *var=value* to provide a non-Boolean value. If *value* contains spaces, it may be surrounded by double quotes. In that case, surround the argument in single quotes to ensure that the double quotes are recorded in the property::
+  Append the value of *argument* to the kernel command line. The value of *argument* can be a simple variable name for
+  Boolean arguments or may take the form *var=value* to provide a non-Boolean value. Multiple command-line arguments may
+  be accumulated into a single *argument*. If the value of any variable value contains spaces, it should be surrounded
+  by double quotes. In that case, surround the entire argument in single quotes to ensure that the double quotes are
+  recorded in the property::
 
     zbm-kcl -a 'variable="some argument with spaces"'
 
   This argument may be repeated any number of times.
 
 **-r** *argument*
 
-  Remove *argument* from the kernel command line. The value of *argument* can be a simple variable name, in which case all arguments of the form *argument* or *argument=<arbitrary-value>* will be stripped. Alternatively, a specific argument may be selected by specifying *argument=<specific-value>*.
+  Remove *argument* from the kernel command line. The value of *argument* can be a simple variable name, in which case
+  all arguments of the form *argument* or *argument=<arbitrary-value>* will be stripped. Alternatively, a specific
+  argument may be selected by specifying *argument=<specific-value>*.
 
   This argument may be repeated any number of times.
 
   .. note::
 
-    All removal options are processed *before* any append options are processed, making it possible to replace an existing argument by combining removal and append options into a single invocation of **zbm-kcl**.
+    All removal options are processed *before* any append options are processed, making it possible to replace an
+    existing argument by combining removal and append options into a single invocation of **zbm-kcl**.
 
 **-e**
 
-  Open the contents of the command-line property in an interactive editor. If the environment defines *$EDITOR*, that will be used; otherwise, **vi** will be used by default. After making changes as desired, overwrite the (temporary) file that was opened and quit the editor. If the contents of the command-line property appear to have changed, **zbm-kcl** will apply those changes.
+  Open the contents of the command-line in an interactive editor. If the environment defines *$EDITOR*, that will be
+  used; otherwise, **vi** will be used by default. After making changes as desired, overwrite the (temporary) file that
+  was opened and quit the editor. If the contents of the command-line property appear to have changed, **zbm-kcl** will
+  apply those changes.
 
```

Comment:
You're right, this used to be the case but, because `zbm-kcl` now allows appends and removals before editing, it is too much work to figure out whether the contents of the KCL have changed. Editing mode *always* rewrites the KCL on exit. I'll correct the documentation here.